frontend/src/metabase/visualizations/echarts/pie/constants.ts

```diff
@@ -36,8 +36,8 @@ export const OTHER_SLICE_MIN_PERCENTAGE = 0.005;
 
 export const OTHER_SLICE_KEY = `${NULL_CHAR}___OTHER___`;
 
-export const OTHER_SLICE_NAME = t`Other`;
+export const getSliceName = () => t`Other`;
```

Comment:
```suggestion
export const getOtherSliceName = () => t`Other`;
```
Was the removal of "other" intentional?

Reply:
Aweeeee sheesh. That was a mistaken. Thanks for spotting it. 🙏

Reply:
Fixed in https://github.com/metabase/metabase/pull/49868/commits/9549a7065e1715fdbc487eb46de4202af2e8737e